sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret.hpp

```diff
@@ -48,8 +48,8 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
      * @param name The name of the secret.
      * @param value The name of the secret.
      */
-    KeyVaultSecret(std::string name, std::string value)
-        : Name(std::move(name)), Value(std::move(value))
+    KeyVaultSecret(std::string const& name, std::string const& value)
+        : Name(std::move(name)), Value(std::move(value)), Properties(std::move(name))
```

Comment:
I think we would not call std::move() when we are taking a const& as the arg.

Also, I wonder what is happening here when calling std::move twice from `name`